packages/notifications/src/hooks/use-session.ts

```
@@ -0,0 +1,54 @@
+'use client';
+
+import { useState, useEffect } from 'react';
+import OneSignal from 'react-onesignal';
+
+export function useSession() {
+  /**
+   * @brief Links current client session to a unique user
+   */
+  const linkToUser = (userId: string) => {
+    const [isLoading, setIsLoading] = useState(true);
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)